controllers/users.js

```diff
@@ -0,0 +1,132 @@
+const { userValidator } = require("./../utils/validators/validator");
+const service = require("../service/users");
+const jwt = require("jsonwebtoken");
+const User = require("../service/schemas/user");
+require("dotenv").config();
```

Comment:
Gdyby na przykład grupować importy tutaj wyglądałoby to mniej więcej tak: 
```suggestion
require("dotenv").config();
const jwt = require("jsonwebtoken");

const User = require("../service/schemas/user");
const service = require("../service/users");
const { userValidator } = require("./../utils/validators/validator");
```